### PR TITLE
Add ingress for the app

### PR DESCRIPTION
**Pull request by QuentinBisson**
@webwurst and @ced0ps could i get your review here ?

### CODE REVIEW

**Review by webwurst — APPROVED** [review]
LGTMIIWFY

File: helm/oauth2-proxy-app/templates/ingress.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{{- if .Values.ingress.enabled }}
+apiVersion: networking.k8s.io/v1beta1
```
Comment (ubergesundheit) [inline helm/oauth2-proxy-app/templates/ingress.yaml:2]: Please use the new one!

**Comment by webwurst** [comment]
Maybe `protocol` needs to be `tcp` anyway?

**Review by ubergesundheit — CHANGES_REQUESTED** [review]
IMO we should not release something with soon to be removed api versions. If you really need the old version, implement a apiversion switch

**Comment by webwurst** [comment]
Also port could stay at `4180`. And only the `targetPort` needs to be configurable?

**Comment by QuentinBisson** [comment]
You are right @webwurst  :)

**Comment by QuentinBisson** [comment]
The latest commit should make @ubergesundheit happy :D

File: helm/oauth2-proxy-app/templates/ingress.yaml (hunk 1 above)
Comment (QuentinBisson) [reply helm/oauth2-proxy-app/templates/ingress.yaml:5]: Could you give it another look ?

**Comment by QuentinBisson** [comment]
Tested on Gauss with https://github.com/giantswarm/config/pull/494

**Review by ubergesundheit — APPROVED** [review]

**Pull request merged**